backend/src/users/user/user.controller.ts

```diff
@@ -95,7 +94,6 @@ export class UserController {
 		});
 	}
 
-	@UseGuards(AuthGuard)
```

Comment:
This probably still needs the guard right? Or do we want to be able to update a user without being authenticated?

Reply:
Yes thank you miss this one!